api/v3/Case.php

```diff
@@ -101,31 +131,46 @@ function civicrm_api3_case_create($params) {
     CRM_Case_BAO_CaseContact::create($contactParams);
   }
 
+  if (!isset($params['id'])) {
+    // Creating a new case
```

Comment:
Is this comment correct? Are we really creating a case here? If not, please use the comment to explain why do we need this